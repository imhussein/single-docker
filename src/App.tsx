import React, { Component } from "react";

interface AppComponentProps {}

interface AppComponentState {}

class App extends Component<AppComponentProps, AppComponentState> {
  constructor(props: AppComponentProps) {
    super(props);
    this.state = {};
  }

  render(): JSX.Element {
    return (
      <div className="App">
        <p className="Apptext">App</p>
      </div>
    );
  }
}

export default App;
