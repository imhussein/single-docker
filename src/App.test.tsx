import React from "react";
import { render } from "@testing-library/react";
import App from "./App";
import "./config";
import { shallow } from "enzyme";
let appComponent = shallow(<App />);

describe("Render App Component", () => {
  it("Match App Component Snapshot", () => {
    expect(appComponent).toMatchSnapshot();
  });

  it("App Component Text", () => {
    expect(appComponent.find(".Apptext").text()).toBe("App Component Changed");
  });
});
