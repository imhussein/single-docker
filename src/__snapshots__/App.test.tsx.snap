// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App Component Match App Component Snapshot 1`] = `ShallowWrapper {}`;
